api/suite_test.go

```diff
@@ -97,6 +97,8 @@ func (s *S) SetUpSuite(c *check.C) {
 	app.AuthScheme = nativeScheme
 	p := app.Platform{Name: "zend"}
 	s.conn.Platforms().Insert(p)
```

Comment:
cool! I'll move this to the specific test, it makes more sense to add the platform just there.